app/translate_survey.py

```diff
@@ -258,8 +258,14 @@ def translate_introduction_content(container, context, translations):
 
 def translate_definitions_text(container, context, translations):
     if 'definitions' in container:
-        for definitions in container['definitions']:
-            translate_container(definitions, context, translations)
+        for definition in container['definitions']:
+            translate_container(definition, context, translations)
+
+            for item in definition['content']:
```

Comment:
Maybe we should define a common function to extract from a "content" type? seeing as we use the pattern in multiple places?